### PR TITLE
Add pushgateway

### PR DESCRIPTION
**Pull request by mdebruyn-trip**

### CODE REVIEW

File: manifests/pushgateway.pp [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+#  [*service_name*]
+#  Name of the node exporter service (default 'pushgateway')
+#
+#  [*user*]
+#  User which runs the service
+#
+#  [*version*]
+#  The binary release version
+class prometheus::pushgateway (
+  $arch                 = $::prometheus::params::arch,
```
Comment (bastelfreak) [inline manifests/pushgateway.pp]: Hi, please add datatypes to this class. an example can be found in https://github.com/voxpupuli/puppet-prometheus/pull/46

**Comment by bastelfreak** [comment]
Hi @mdebruyn-trip, thanks for this PR! Please take a look at the inline comment

**Comment by mdebruyn-trip** [comment]
I've pushed the changes to add datatypes and addressed the failing tests.  Let me know if you need anything else.  Thanks.

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**